cosmos-sdk-proto/Cargo.toml

```diff
@@ -10,16 +10,16 @@ readme     = "README.md"
 categories = ["cryptography", "cryptography::cryptocurrencies", "database"]
 keywords   = ["blockchain", "cosmos", "tendermint", "proto"]
 
-
 [dependencies]
 prost = "0.6"
 prost-types = "0.6"
-tonic = "0.3.1"
-git2 = "0.13"
-prost-build = "0.6"
-tempdir = "0.3"
-walkdir = "2"
-tonic-build = "0.3.1"
```

Comment:
These appear to be superfluous (moved to `proto-build`?)

Reply:
yes they are for proto-build, good catch.